lib/dialogs/logout-confirmation/index.tsx

```diff
@@ -32,56 +34,45 @@ export class LogoutConfirmation extends Component<Props> {
 
     return (
       <div className="logoutConfirmation">
-        <Dialog onDone={closeDialog} title="Unsynchronized Notes Found">
-          <section>
-            <h1>Are you sure you want to logout?</h1>
-          </section>
-
+        <Dialog onDone={closeDialog} title="Unsynced Notes">
+          <p className="explanation">
+            {notes.size > 0
+              ? 'Logging out will delete any unsynced notes.'
+              : 'All notes have syncronized!'}
+          </p>
+
+          <p className="explanation-secondary">
+            {notes.size > 0 && 'Possibly unsynchronized notes'}
+          </p>
```

Comment:
Do we need an empty paragraph if there are no changes?

Reply:
fixed it